Build/Projects/PipelineReferences.definition

```diff
@@ -2,6 +2,7 @@
 <ExternalProject Name="PipelineReferences">
 
   <Platform Type="Linux">
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-preview.2" />
```

Comment:
Only needed if targeting .NET 4.7.0 or lower.

Reply:
We target 4.7.2 now, so you can't target lower. Mono does not yet support 4.7.2, that's why the reference assemblies are required.

Reply:
> We target 4.7.2 now, so you can't target lower.

Yes, with 4.7.2 the package is redundant.

> Mono does not yet support 4.7.2, that's why the reference assemblies are required.

It does support it, it supported it for a long time.

Reply:
I'm on Ubuntu 18.04, which does not have a new enough version of mono in it's package repo. That means you can't build without the reference assemblies.